### PR TITLE
blocked cookies message

### PR DESCRIPTION
**Pull request by akash1810**
Adds a message for self diagnosis of blocked cookies.

screen shot 2015-11-24 at 15 45 57

### CODE REVIEW

**Comment by akash1810** [comment]
the image is a little crude...

File: kahuna/public/js/errors/global.html [hunk 1]
```diff
@@ -21,6 +26,11 @@
                target="_self" class="coloured-link">login again</a>
             to reauthenticate.
         </div>
+
+        <p>
+            If you can see this icon <img src="/assets/images/blocked-cookies.png" alt="blocked third party cookies" class="side-padded"/> in the address bar,
+            please follow <a href="https://docs.google.com/a/guardian.co.uk/document/d/1qlQhHM2NX3xWBIREB2O8D7S4ZtN0sYb7IDV2leE4Ubc/edit?usp=sharing" class="coloured-link" target="_blank">these instructions</a>.
```
Comment (theefer) [inline kahuna/public/js/errors/global.html]: Looks like you have to have a guardian.co.uk email to access this. Could you request it to be public for all instead?

**Comment by theefer** [comment]
:+1:

**Comment by theefer** [comment]
Nice!

File: kahuna/public/js/errors/global.html (hunk 1 above)
Comment (akash1810) [inline kahuna/public/js/errors/global.html]: > Could you request it to be public for all instead?

@jamesgorrie I think you're the owner of the doc, can you change the permissions? I can't work out how to request access level changes from within Google Docs :cry:

Comment (jamesgorrie) [inline kahuna/public/js/errors/global.html]: Done.

**Comment by paperboyo** [comment]
Hello,

Spaghetti Monster my witness that I’ve tried to create my first-ever PR, but git web interface does not allow uploading binary files, so maybe another time...

Here is an original Google’s blocked cookies icon, just optimised a bit better:

blocked_cookies_128colours

cookies icon before-after
[left: proposed (1,299 bytes); right: current (6,717 bytes)]

Regards
Mateusz

**Pull request merged**